devices/dim_light.py

```diff
@@ -0,0 +1,25 @@
+import Domoticz
+from devices.device import Device
+
+class DimLight(Device):
```

Comment:
What is the difference between `DimLight` and `DimmerSwitch`, for me they look very similar, can we just leave one of them and move the difference to `convert_message` method of corresponding adapter?

Reply:
I think it will be good to keep the difference since they represent different devices: a dimmer can (basically) only send commands (user hits keys etc) while a light can act on commands )(switch on/off etc). In the end the message coming back is the same (status update from the light).

But at this moment the differences ain't that big indeed. But I would like to propose to keep it in.

Reply:
My idea is to keep Domoticz devices in `devices` folder and put hardware specific details (custom message format, conversions, handling commands) to its adapter. In this case devices can be reused by different adapters and there should help avoid mess once amount of adapters continue increasing